demo/src/index.tsx

```diff
@@ -13,12 +13,20 @@ const data = [
     heading: 'A Table',
     child: <table><tbody><tr><td>Hello</td></tr></tbody></table>,
     open: true
+  },
+  {
+    name: 'A Third Option',
+    heading: 'A Third Option',
+    child: <span>Just a third option</span>,
   }
 ];
 
 const DemoPage = () => {
   return (
-    <ReactAccordion items={data}/>
+    <>
+      <ReactAccordion items={data} />
+      <SingleItemOpenAccordion items={data} />
+    </>
```

Comment:
Can we give some headings to the demo page?

```suggestion
    <>
      <h1>ReactAccordion Demo</h1>
      <h2>Accordion that allows multiple open</h2>
      <ReactAccordion items={data} />
      
      <h2>Accordion that only allows one open</h2>
      <SingleItemOpenAccordion items={data} />
    </>
```